From 2df66d04b60abb7dc6ce9a3ae44890472291201f Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:10 -0700
Subject: [PATCH] cavium: fix printk_ratelimit

Correct the use of __printk_ratelimit(), which has been removed from recent kernels.

Signed-off-by: Tomaso Paoletti <tpaoletti@caviumnetworks.com>
---
 arch/mips/cavium-octeon/ethernet-pow.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/cavium-octeon/ethernet-pow.c b/arch/mips/cavium-octeon/ethernet-pow.c
index 6f6e132..5a5bccb 100644
--- a/arch/mips/cavium-octeon/ethernet-pow.c
+++ b/arch/mips/cavium-octeon/ethernet-pow.c
@@ -21,7 +21,7 @@
 #define IP_PROTOCOL_TCP 6	/* IP header protocol for TCP */
 #define IP_PROTOCOL_UDP 0x11	/* IP header protocol for UDP */
 
-#define DEBUGPRINT(format, ...) do{if (__printk_ratelimit(HZ, 10)) printk(format, ##__VA_ARGS__);} while (0)
+#define DEBUGPRINT(format, ...) do{if (printk_ratelimit()) printk(format, ##__VA_ARGS__);} while (0)
 
 /* These are the Octeon CSR IO addresses we are going to need */
 #define OCTEON_POW_WQ_INT_THRX(offset)  (0x8001670000000080ull+((offset)*8))
-- 
1.5.5.1

